modules/ROOT/pages/6.0-release-notes-known-issues.adoc

```diff
@@ -23,4 +23,20 @@ As of this release, however, the specific cause is still being investigated.
 Workaround::
 Other than keeping `nonbreaking_force_tab` set to `false`, no workaround is currently available.
 
-// end::known-issues[]
\ No newline at end of file
+== Editor content can overlap with an open floating toolbar drawer when using certain premium skins
+
+In {productname} 6.0, if
+
+* the floating toolbar drawer is open; and
+* one of the following premium skins is being used
++
+— `snow`, `naked`, `outside`, or `fabric` —
+
+content within the {productname} editor may overlap visually with the toolbar drawer.
```

Comment:
```suggestion
In {productname} 6.0, content within the {productname} editor may overlap visually with the toolbar drawer if

* the floating toolbar drawer is open; and
* one of the following premium skins is being used
+
— `snow`, `naked`, `outside`, or `fabric` —
```
Figure it removes at least one of our problems to put the last sentence near the beginning. Unsure what to do with the skin names, but either moving them into the line above or using Brian's nested list idea sounds like it'd work to me.